templates/download/server/download.html

```diff
@@ -1,130 +1,269 @@
+{% block title %}Thank you for downloading Ubuntu Server{% endblock %}
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock body_class %}
+
 {% block head_extra %}
   <meta name="robots" content="noindex" />
 {% endblock %}
 
 {% block content %}
-  <noscript>
-    <meta
-      http-equiv="refresh"
-      content="3;url=https://releases.ubuntu.com/{{ version }}/ubuntu-{{ version }}-live-server-amd64.iso"
-    />
-  </noscript>
-
-  <section class="p-strip u-no-padding--top u-no-padding--bottom" style="overflow: visible;">
-    <div class="p-strip--suru-topped">
+<noscript>
+  <meta
+    http-equiv="refresh"
+    content="3;url=https://releases.ubuntu.com/{{ version }}/ubuntu-{{ version }}-live-server-amd64.iso"
+  />
+</noscript>
+
+<section class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="js-download-option">Thank you for downloading Ubuntu Server {{ version }}</h1>
+      <p>Your download should start in the background. If it doesn't,
+        <a href="https://releases.ubuntu.com/{{ version }}/ubuntu-{{ version }}-live-server-amd64.iso">download&nbsp;now</a>.<br />
+        {% with version=version, system="live-server", architecture="amd64" %}
+          {% include "download/shared/_verify-checksums.html" %}
+        {% endwith %}
+      </p>
+    </div>
+    <div class="col">
+      {% include 'shared/_server-download-newsletter.html' %}
+    </div>
+  </div>
+</section>
+
+{% include 'download/shared/_server_weekly_news.html' %}
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Ubuntu CLI cheatsheet</h2>
+      <div class="p-section--shallow">
+        <p>Learn how to use the command line efficiently and get started with DevOps — from basic file management to LXD virtualisation and Ubuntu Pro.</p>
+      </div>
+      <button class="p-button--positive" onclick="toggleModal()">Download the cheatsheet</button>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow p-image-wrapper">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/639c1f12-server-cli-cheatsheet.png",
+          alt="CLI cheat sheet",
+          width="1200",
+          height="849",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row">
+    <div class="col-3 col-medium-2">
+      <div class="p-section">
+        <h2 class="p-muted-heading">Resources</h2>
+      </div>
+    </div>
+    <div class="col-9">
+     <div class="row">
+      <div class="col-3 col-medium-2">
+        <h3 class="p-heading--5"><a href="/server/docs">Ubuntu Server documentation</a></h3>
```

Comment:
There's a different link for this on the doc, but it throws 404 so I assume this is the correct one. Would you mind updating on the doc if you've already confirmed?

Reply:
Done.